src/index.js

```diff
@@ -1,34 +1,42 @@
 import React from 'react';
```

Comment:
the entire App MUST be mounted on the DOM inside the redux store provider otherwise we won't be able to access our redux store